src/protocol/organization.rs

```diff
@@ -175,6 +176,53 @@ impl OrganizationConfigReq {
     }
 }
 
+#[pyclass]
+#[derive(Debug, Clone)]
+pub(crate) struct ActiveUsersLimit(pub types::ActiveUsersLimit);
+
+#[pymethods]
+impl ActiveUsersLimit {
+    #[classmethod]
+    #[pyo3(name = "LimitedTo")]
+    fn limited_to(_cls: &PyType, user_count_limit: u64) -> Self {
+        Self(types::ActiveUsersLimit::LimitedTo(user_count_limit))
+    }
+
+    #[classattr]
+    #[pyo3(name = "NO_LIMIT")]
+    fn no_limit() -> &'static PyObject {
+        lazy_static::lazy_static! {
+            static ref VALUE: PyObject = {
```

Comment:
```suggestion
    fn no_limit() -> &'static Self {
        lazy_static::lazy_static! {
            static ref VALUE: Self = {
```

This can't be type to `Self` ?

Reply:
We can't put `Self` here unfortunately because of `lazy_static`. 
```
error[E0401]: can't use generic parameters from outer function
   --> src/protocol/organization.rs:195:31
    |
194 | /         lazy_static::lazy_static! {
195 | |             static ref VALUE: Self = {
    | |                               ^^^^
    | |                               |
    | |                               use of generic parameter from outer function
    | |                               use a type here instead
196 | |                 Python::with_gil(|py| {
197 | |                     ActiveUsersLimit(types::ActiveUsersLimit::NoLimit)
198 | |                 })
199 | |             };
200 | |         };
    | |_________- `Self` type implicitly declared here, by this `impl`
```

Hardcoding the concrete type doesn't work either.
```
error[E0277]: the trait bound `&protocol::organization::ActiveUsersLimit: IntoPyCallbackOutput<_>` is not satisfied
```
We need to return a reference of a static attribute here otherwise the code will no longer be true in python. I think that's why all static attribute in `src/enumerate.rs` returns `&PyObject` (which implements `IntoPyCallbackOutput`).
```python
a = ActiveUsersLimit.NO_LIMIT
a is ActiveUsersLimit.NO_LIMIT # True
```

Reply:
Maybe use `ActiveUsersLimit` instead of `Self` in the `lazy_static` block :thinking:

Reply:
> Hardcoding the concrete type doesn't work either.

Unfortunately it's not working either. 

```
error[E0277]: the trait bound `&protocol::organization::ActiveUsersLimit: IntoPyCallbackOutput<_>` is not satisfied
   --> src/protocol/organization.rs:183:1
    |
183 | #[pymethods]
    | ^^^^^^^^^^^^ the trait `IntoPyCallbackOutput<_>` is not implemented for `&protocol::organization::ActiveUsersLimit`
```

Reply:
sad :'(